hub/management/commands/import_gbgw_events.py

```diff
@@ -0,0 +1,126 @@
+from time import sleep
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db.models import Avg, IntegerField
+from django.db.models.functions import Cast
+
+import pandas as pd
+from tqdm import tqdm
+
+from hub.models import Area, AreaData, DataSet, DataType
+from utils.mapit import (
+    BadRequestException,
+    ForbiddenException,
+    InternalServerErrorException,
+    MapIt,
+    NotFoundException,
+    RateLimitException,
+)
+
+
+class Command(BaseCommand):
+    help = "Import data about number of GBGW events per constituency"
+
+    data_file = settings.BASE_DIR / "data" / "gbgw_events.csv"
+    column_map = {}
+    data_types = {}
+
+    def get_dataframe(self):
+        df = pd.read_csv(self.data_file)
+        df = df.dropna(axis="columns", how="all")
+
+        return df
+
+    def add_data_sets(self, df):
+        data_set, created = DataSet.objects.update_or_create(
+            name="constituency_gbgw_2022_event_count",
+            defaults={
+                "label": "Number of Great Big Green Week 2022 Events",
+                "description": "Number of Great Big Green Week 2022 Events",
+                "data_type": "integer",
+                "category": "movement",
+                "source_label": "TCC",
+                "source": "",
```

Comment:
I know the data for this isn’t published, but I wonder whether a link to https://greatbiggreenweek.com/ might at least set people on a useful path, rather than giving them nothing?